buildozer/default.spec

```diff
@@ -22,7 +22,7 @@ source.include_exts = py,png,jpg,kv,atlas
 #source.exclude_exts = spec
 
 # (list) List of directory to exclude (let empty to not exclude anything)
-#source.exclude_dirs = tests, bin
+#source.exclude_dirs = tests, bin, venv
```

Comment:
I too often see people doing reports because buildozer picked up some `venv` garbage, so here're some pointers for people who care to read through the default example :smile: